web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx

```diff
@@ -4,10 +4,19 @@ export const getTime = (
   useLocalTimezone: boolean
 ): string => {
   if (!issueWithTimeFormat) {
+    let formatOptions: {} = {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+    }
     if (useLocalTimezone) {
-      return new Date(dateString).toLocaleString(navigator.language)
+      return new Date(dateString).toLocaleString('no-NB', formatOptions)
```

Comment:
Any reason to hard code "norway" as local time here?

Reply:
We don't want to hard code Norway as the local time. Operations can occur all over the world.

Reply:
fixed!